code/gallery.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Gallery</title>
+    <link href="style.css" rel = "stylesheet" type="text/css"> 
+
+  </head>
+<body>
+  <h1>Gallery!</h1>
+    <div style="text-align: center"><img src="/Users/livartda/Documents/project-p1-first-webpage-master/code/SAM_1220.JPG" width="800" height="550"></div>
+    <menu>
+        <a href="/Users/livartda/Documents/project-p1-first-webpage-master/code/index.html">Home</a>
+        <a href="/Users/livartda/Documents/project-p1-first-webpage-master/gallery.html">Gallery</a>
+      </menu>
+      
+      <p class="box-one">
+As a person that likes to travel, I often bring my camera with me to snap some photos of the places I go to.
+<br>
+<br>Below is a few of the photos I have captured during the years while travelling to different places in Europe, but also my home country Norway
+
+<br>If you click on the photos below they will open in a new tab in your browser
+    </p>
+
+
+   <div class="box-one">   
+<div class="gallery">
+  <a target="_blank" href="/Users/livartda/Documents/project-p1-first-webpage-master/code/IMG_2282-1.jpg">
```

Comment:
When you're changing the target to `"_blank"`, you create a potential security issue. To fix, add the attribute `rel="noopener noreferrer"` as well. Read [this article](https://medium.com/sedeo/how-to-fix-target-blank-a-security-and-performance-issue-in-web-pages-2118eba1ce2f) if you want to know more :)

Reply:
Thank you for explaining this, did not know this could cause security issues but after reading the article, I understand how e.g phishing might have happend.